schemas/system_profile/v1.yaml

```
@@ -650,3 +650,9 @@ $defs:
         maxLength: 10
         enum: [dnf, rpm-ostree, yum]
         example: "dnf, rpm-ostree, yum"
+      virtual_host_uuid:
+        description: hypervisor host identity (subscription manager id)
+        type: string
+        pattern: '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
```

Looking elsewhere in this schema file, match patterns for hex numbers are somewhat inconsistent (MAC addresses match on both cases, but other UUIDs only check lowercase), but my paranoid side thinks the UUID patterns should include matches on uppercase even though I don't know if they will ever be generated that way. 🤷

```suggestion
        pattern: '[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}'
```

---

As discussed on Slack, let's just remain consistent with the other UUID patterns and expect the reporters to set the format consistently.